packages/firebase_ai/firebase_ai/example/lib/pages/function_calling_page.dart

```diff
@@ -170,23 +170,21 @@ class _FunctionCallingPageState extends State<FunctionCallingPage> {
     final functionCalls = response.functionCalls.toList();
     // When the model response with a function call, invoke the function.
     if (functionCalls.isNotEmpty) {
-      final functionCall = functionCalls.first;
-      if (functionCall.name == 'fetchWeather') {
-        Map<String, dynamic> location =
-            functionCall.args['location']! as Map<String, dynamic>;
-        var date = functionCall.args['date']! as String;
-        var city = location['city'] as String;
-        var state = location['state'] as String;
-        final functionResult = await fetchWeather(Location(city, state), date);
-        // Send the response to the model so that it can use the result to
-        // generate text for the user.
-        response = await functionCallChat.sendMessage(
-          Content.functionResponse(functionCall.name, functionResult),
-        );
-      } else {
-        throw UnimplementedError(
-          'Function not declared to the model: ${functionCall.name}',
-        );
+      for (final functionCall in functionCalls) {
```

Comment:
This looks like a fine change to make - how does it relate to the rest of the PR? Consider comiting separately.